src/shared/download-file.ts

```diff
@@ -19,25 +19,6 @@ function downloadFile(
   URL.revokeObjectURL(url);
 }
 
-/**
- * Download a file containing JSON-serialized `object` as `filename`
- *
- * @param data - JSON-serializable object
- * @return The contents of the downloaded file
- * @throws {Error} If provided data cannot be JSON-serialized
- */
-export function downloadJSONFile(
-  data: object,
-  filename: string,
-  /* istanbul ignore next - test seam */
-  _document = document,
-): string {
-  const fileContent = JSON.stringify(data, null, 2);
-  downloadFile(fileContent, 'application/json', filename, _document);
-
-  return fileContent;
-}
-
```

Comment:
Refactored this function so that it expects data as `string` instead of as `object`.

That means the JSON.stringify call is delegated to calling code, which implies they could potentially pass a non-JSON string, but has the benefit that it makes the process of downloading or copying to clipboard more consistent and removes complexity.

We discussed this approach in slack https://hypothes-is.slack.com/archives/C1M8NH76X/p1705312914535639